README.md

```diff
@@ -214,8 +251,6 @@ are pushed to the Arkouda or Arkouda fork _master branch_. Next, navigate to the
 option. The Github Pages docs url will be displayed once the source option is selected. Click on the link and the
 Arkouda documentation homepage will be displayed.
 
-</details>
```

Comment:
Was this deleted intentionally? It looks like we just have an open details tag there now.

Reply:
It was, but it was the wrong fix. Found where a comment block was improperly configured and fixed.